packages/backend/discovery/zora/ethereum/config.jsonc

```diff
@@ -28,6 +28,7 @@
       "ignoreInWatchMode": ["params"]
     },
     "L2OutputOracle": {
+      "extends": "opstack/L2OutputOracle",
```

Comment:
Something snuck it's way into this PR :p

Reply:
XD, good catch!